package.json

```diff
@@ -3,27 +3,19 @@
   "version": "0.0.0",
   "private": true,
   "license": "Apache-2.0",
-  "packageManager": "yarn@1.22.19",
-  "workspaces": {
```

Comment:
We don't need this anymore? I think perhaps lerna relies on it.

Reply:
not sure - lerna is used for publishing packages only right? I can keep this in here, but it seemed like they were `yarn` configurations. I wonder if since I configured lerna to `pnpm` it will look at the `pnpm-workspace.yaml` instead

Reply:
@TooTallNate is correct, `lerna` still needs `workspaces` in the `package.json`.

```
$ lerna changed
lerna notice cli v6.0.3
lerna ERR! EWORKSPACES Workspaces need to be defined in the root package.json.
lerna ERR! EWORKSPACES See: https://github.com/lerna/lerna/blob/master/commands/bootstrap/README.md#--use-workspaces
```

Reply:
It seems lerna _should_ be able to use pnpm-workspace.yaml https://lerna.js.org/docs/recipes/using-pnpm-with-lerna investigating now

Reply:
Ahh it must only support it in like v5+ - we are using v3 :(

Reply:
Can we update Lerna? The last v3 release was 2-1/2 years ago. Lerna (again) is being maintained and it makes most sense to keep current. I'm not sure how the release process would be affected.

Reply:
imo we should migrate off lerna. use pnpm+turbo instead

Reply:
I can try updating it now to atleast v5 

v6 uses Nx lol

Reply:
Sigh, figures. +1 for pnpm+turbo. Are there any projects out there that uses pnpm+turbo for publishing/changelog that we can look to for inspiration?

Reply:
https://github.com/vercel/turbo lol they've got a whole changeset workflow we can clone

Reply:
oh nvm they moved to something else: https://github.com/vercel/turbo/blob/main/.github/workflows/publish-npm.yml

but we should still consult them.

Reply:
yay lerna5 works! try it out locally and let me know if you hit some edge case I didn't